tests/tests.js

```diff
@@ -175,3 +175,12 @@ test.skip('works with very large lengths', function (t) {
 	t.end();
 });
 */
+
+test('returns the correct name when called with toString', function (t) {
```

Comment:
this test will fail in IE, which doesn't have a function "name" property. You'll want to skip this test (using `{ skip: condition }` as the second argument to `test`) when a runtime detection for function name support fails. See https://github.com/es-shims/String.prototype.trim/blob/master/test/shimmed.js#L16-L19